frontend/src/pages/admin/project-list.js

```diff
@@ -44,216 +60,177 @@ function projectToRow (project, onDeleteClick) {
-                  component={(props: any) => <Link {...props} to="/admin/projects/new" />}
-                >
-                  <PlusCircleIcon /> Add Project
-                </Button>
-              </FlexItem>
-            </Flex>
+  document.title = 'Projects - Administration | Ibutsu';
+  return (
+    <React.Fragment>
+      <PageSection id="page" variant={PageSectionVariants.light}>
+        <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+          <Flex>
+            <FlexItem spacer={{ default: 'spacerLg' }}>
+              <TextContent>
+                <Text className="title" component="h1" ouiaId="admin-projects">Projects</Text>
+              </TextContent>
+            </FlexItem>
           </Flex>
-        </PageSection>
-        <PageSection className="pf-u-pb-0">
-          <Card>
-            <CardBody className="pf-u-p-0">
-              <FilterTable
-                columns={columns}
-                rows={rows}
-                filters={filters}
-                pagination={pagination}
-                isEmpty={this.state.isEmpty}
-                isError={this.state.isError}
-                onSetPage={this.setPage}
-                onSetPageSize={this.setPageSize}
-              />
-            </CardBody>
-          </Card>
-        </PageSection>
-        <Modal
-          title="Confirm Delete"
-          variant="small"
-          isOpen={this.state.isDeleteModalOpen}
-          onClose={this.onDeleteModalClose}
-          actions={[
-            <Button
-              key="delete"
-              variant="danger"
-              ouiaId="admin-projects-modal-delete"
-              isLoading={this.state.isDeleting}
-              isDisabled={this.state.isDeleting}
-              onClick={this.onModalDeleteClick}
-            >
-              {this.state.isDeleting ? 'Deleting...' : 'Delete'}
-            </Button>,
-            <Button
-              key="cancel"
-              variant="secondary"
-              ouiaId="admin-projects-modal-cancel"
-              isDisabled={this.state.isDeleting}
-              onClick={this.onDeleteModalClose}
-            >
-              Cancel
-            </Button>
-          ]}
-        >
-          Are you sure you want to delete &ldquo;{this.state.selectedProject && this.state.selectedProject.title}&rdquo;? This cannot be undone!
-        </Modal>
-      </React.Fragment>
-    );
-  }
-}
+          <Flex>
+            <FlexItem>
+              <Button
+                aria-label="Add project"
+                variant="secondary"
+                title="Add project"
+                ouiaId="admin-projects-add"
+                component={(props) => <Link {...props} to="/admin/projects/new" />}
+              >
+                <PlusCircleIcon /> Add Project
+              </Button>
+            </FlexItem>
+          </Flex>
+        </Flex>
+      </PageSection>
+      <PageSection className="pf-u-pb-0">
+        {!isEmpty &&
```

Comment:
We should check here if `projects.length > 0` instead of isEmpty. So if there are no projects at all - we will show EmptyObject with `No Projects found` & `Create your first project`. But if length is more than 0 - we have some projects and now we will show filteredProjects. If it's going to be empty list (in case we used filter - isEmpty will take care of showing table empty thing)

isEmpty is used as param for FilterTable and when it's set to true - table shows No results found
![image](https://github.com/user-attachments/assets/53f681fe-bb33-42c8-8bce-7e0fbdcfc48f)

Reply:
Updated to use FilterTable empty state here, and kept the custom empty state when no projects are found at all.